src/App.jsx

```diff
@@ -1,104 +1,20 @@
 import './App.scss';
+import { PostList } from './components/PostList';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+
+const postsWithComments = postsFromServer.map(el => {
+  return {
+    ...el,
+    comments: commentsFromServer.filter(comment => comment.postId === el.id),
+  };
+});
```

Comment:
Ensure that the mapping logic for `postsWithComments` correctly aligns with the task requirements. Verify that each post is correctly associated with its comments based on `postId`. This logic assumes that `commentsFromServer` is an array of comment objects with a `postId` property, and `postsFromServer` is an array of post objects with an `id` property. If the data structure is different, this logic may need adjustment.